Makefile

```
@@ -56,4 +73,20 @@ test-release: check-bb
+run-ide:
+ifeq ($(OS),Windows_NT)
+	@powershell -Command "Remove-Item -Force -ErrorAction SilentlyContinue '$(IDEA_LOG)'"
+	cmd //c ".\\gradlew.bat runIde"
+else
+	@rm -f $(IDEA_LOG)
+	./gradlew runIde
+endif
+
 iter: format format-check class-size-mine delta test
+
+logs:
```

checkmake (0.3.2)

[warning] Target "logs" should be declared PHONY.

(phonydeclared)